src/Workspaces/Core/Portable/ExternalAccess/UnitTesting/Api/UnitTestingSessionWithSolutionWrapper.cs

```diff
@@ -6,6 +6,8 @@
 using System.Threading.Tasks;
 using Microsoft.CodeAnalysis.Remote;
 
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
📝 Same applies here

Reply:
did this need to be addressed?

Reply:
As I mentioned, we don't have a good replacement for LUT yet, so I'd rather not report bunch of obsolete warnings in their build until we have a good replacement.